packages/autorest.typescript/test/rlcIntegration/generated/multipleUrlParameters/src/parameters.ts

```diff
@@ -38,14 +38,14 @@ export interface EntityListByGuidsPathParameters {
 }
 
 export interface EntityListByGuidsQueryParamProperties {
-  /** An array of GUIDs of entities to list. */
-  guid: Array<string>;
+  /** An array of GUIDs of entities to list.This type has special collectionFormat, we provide buildMultiCollection from serializeHelper.ts to help */
```

Comment:
This parameter needs to be formated as multi collection. we provide buildMultiCollection from serializeHelper.ts to help